package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -1,8 +1,6 @@
 #!/bin/sh /etc/rc.common
 # Copyright (C) 2007-2012 OpenWrt.org
 
-. /lib/functions.sh
```

Comment:
Can you move this in service_start? right up network.sh?

Reply:
I don’t think that’s necessary. As was pointed out, it gets included from `/etc/rc.common`:

https://github.com/openwrt/openwrt/blob/e1aaa1defd2340be3544dc614f905795b4d52f81/package/base-files/files/etc/rc.common#L4